common-app-lib/resources/ops-humanizers.json

```diff
@@ -1,331 +1,2419 @@
-{"type": "alias", "field": "instrument", "source_value": "SURFACE ACOUSTIC WAVE HYGROMETER", "replacement_value": "Surface Acoustic Wave Hygrometer", "reportable": true },
-{"type": "alias", "field": "instrument", "source_value": "TDR (TIME DOMAIN REFLECTOMETRY PROBE)", "replacement_value": "Time Domain Reflectometry Probe", "reportable": true },
-{"type": "alias", "field": "instrument", "source_value": "TEMPERATURE PROBE", "replacement_value": "Temperature Probe", "reportable": true },
-{"type": "alias", "field": "instrument", "source_value": "TEMPERATURE SENSOR", "replacement_value": "Temperature Sensor", "reportable": true },
-{"type": "alias", "field": "instrument", "source_value": "TERRA (MORNING EQUATORIAL CROSSING TIME SATELLITE)", "replacement_value": "Terra", "reportable": true },
-{"type": "alias", "field": "instrument", "source_value": "THEMATIC MAPPER SIMULATOR", "replacement_value": "Thematic Mapper Simulator", "reportable": true },
-{"type": "alias", "field": "instrument", "source_value": "THERMISTOR", "replacement_value": "Thermistor", "reportable": true },
-{"type": "alias", "field": "instrument", "source_value": "THERMOCOUPLE", "replacement_value": "Thermocouple", "reportable": true },
-{"type": "alias", "field": "instrument", "source_value": "THERMOMETER", "replacement_value": "Thermometer", "reportable": true },
-{"type": "alias", "field": "instrument", "source_value": "TM (THEMATIC MAPPER)", "replacement_value": "Thematic Mapper", "reportable": true },
-{"type": "alias", "field": "instrument", "source_value": "TOMS (TOTAL OZONE MAPPING SPECTROMETER)", "replacement_value": "Total Ozone Mapping Spectrometer", "reportable": true },
-{"type": "alias", "field": "instrument", "source_value": "TOPEX ALTIMETER", "replacement_value": "Altimeter", "reportable": true },
-{"type": "alias", "field": "instrument", "source_value": "TOPEX MICROWAVE RADIOMETER", "replacement_value": "Topex Microwave Radiometer", "reportable": true },
-{"type": "alias", "field": "instrument", "source_value": "TOTAL PRECIPITATION SENSOR (HOTPLATE)", "replacement_value": "TPS (Hotplate)", "reportable": true },
-{"type": "alias", "field": "instrument", "source_value": "TOTAL SKY IMAGER", "replacement_value": "Total Sky Imager", "reportable": true },
-{"type": "alias", "field": "instrument", "source_value": "UV OZONE DETECTOR", "replacement_value": "UV Ozone Detector", "reportable": true },
-{"type": "alias", "field": "instrument", "source_value": "WATER LEVEL GAUGE", "replacement_value": "Water Level Gauge", "reportable": true },
-{"type": "alias", "field": "instrument", "source_value": "WIDE FIELD CAMERA", "replacement_value": "Wide Field Camera", "reportable": true },
-{"type": "alias", "field": "instrument", "source_value": "WIND PROFILER", "replacement_value": "Wind Profiler", "reportable": true },
-{"type": "alias", "field": "instrument", "source_value": "WIND SCATTEROMETER", "replacement_value": "Wind Scatterometer", "reportable": true }]
+[
```

Comment:
Dana wanted the same file for all environments. So for now I just replaced the OPS file.  I will just be ingesting the OPS humanizers to all of the environments.  I know the other files will need to be cleaned up.